### DISCUSSION TITLE
firewall + extention

### DISCUSSION THREAD

**Discussion by sdsdsdsrj4rjj** [3kh0/ext-remover#1384] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
anyway to use these exploits for the extension "Classroom relay" which has more extensions in it like "lightspeed alert agent", and "lightspeed filter agent" and also there's a second blocker/firewall which is Fortiguard. anyway around this?

**Comment by thatboyo** [comment]
use a proxy or a mobile hotspot

**Reply by ghost** [reply]
use extension hanger , ltmeat flood , Ltmean print , Dexifer , rigtools.

**Reply by Perplexedykwhat** [reply]
yeah but the network/firewall is still going to block webpages?

**Reply by Charlieis1nsan3** [reply]
use blobwifi, and exthanger. so it bypasses extensions, and network

**Reply by Luigi-Master** [reply]
> yeah but the network/firewall is still going to block webpages?

Then you are better off with a proxy

**Reply by sadgimkitgamer** [reply]
rigtools doesnt work anymore and so doesn't ltmeat flood

**Reply by Charlieis1nsan3** [reply]
> rigtools doesnt work anymore and so doesn't ltmeat flood
maybe for you

**Reply by sdsdsdsrj4rjj** [reply]
> use blobwifi, and exthanger. so it bypasses extensions, and network

What exploit in exthanger?

**Reply by sdsdsdsrj4rjj** [reply]
my chromebook's version is 130.0.6723.126

**Reply by Charlieis1nsan3** [reply]
then you can use ext hanger. its literally in this repository.

**Reply by sdsdsdsrj4rjj** [reply]
it freezes the chromebook

**Reply by Luigi-Master** [reply]
> it freezes the chromebook

That is meant to happen, The first freeze takes a minute because it's hanging the extension and the second freeze is after you press the kill extension that only lasts for like 10-20 seconds.

**Reply by Charlieis1nsan3** [reply, marked as answer]
> it freezes the chromebook

once its done you go to chrome://extensions -_-, I think that's self explanitory
